README.md

```diff
@@ -159,24 +162,19 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 #### IDEs
 
 * [Android Studio](https://developer.android.com/studio/index.html) - The official IDE for Android, based on Intellij IDEA. [![Open-Source Software][OSS Icon]](http://tools.android.com/) ![Freeware][Freeware Icon]
-* [AppCode](https://www.jetbrains.com/objc/) - Smart IDE for iOS/macOS development
-* [BBEdit](http://www.barebones.com/products/bbedit/) - The leading professional HTML and text editor for Mac.
-* [Brackets](http://brackets.io) - Modern, open-source IDE for web design by Adobe. [![Open-Source Software][OSS Icon]](https://github.com/adobe/brackets/) ![Freeware][Freeware Icon]
-* [CLion](https://www.jetbrains.com/clion/) - Powerful C and C++ IDE. (**Free** for Students)
-* [Coda2](http://panic.com/coda/) - Fast, clean and powerful text editor.
-* [Deco IDE](https://www.decosoftware.com/) - The best IDE for building React Native apps. [![Open-Source Software][OSS Icon]](https://github.com/decosoftware/deco-ide) ![Freeware][Freeware Icon]
+* [Deco IDE](https://www.decoide.org) - The best IDE for building React Native apps. [![Open-Source Software][OSS Icon]](https://github.com/decosoftware/deco-ide) ![Freeware][Freeware Icon]
 * [Eclipse](https://www.eclipse.org) - Popular open-source IDE, mainly for Java but with plugin support for a wide array of languages and platforms. ![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]
 * [Espresso](http://espressoapp.com/) - The web editor for Mac is back. For people who make delightful, innovative and fast websites.
-* [IntelliJ IDEA](https://www.jetbrains.com/idea/) - Powerful Java IDE. (**Free** for Students)
+* [JetBrains Toolbox App](https://www.jetbrains.com/toolbox/) - Manage installed JetBrains tools, download new ones and open recent projects
+  * [AppCode](https://www.jetbrains.com/objc/) - Smart IDE for iOS/macOS development
```

Comment:
Spacing issues here

Reply:
Hmm.. what do you mean? It's a nested list if you are about the first space before the star

Reply:
![image](https://user-images.githubusercontent.com/16080436/42834908-de99c15a-89c5-11e8-89ae-83047a6761dc.png)